tests/testthat/test-install-torsten.R

```diff
@@ -0,0 +1,157 @@
+## Adapted from test-install.R from the cmdstanr package
+
+context("install_torsten")
+
+torsten_test_tarball_url <- Sys.getenv("TORSTEN_TEST_TARBALL_URL")
+if (!nzchar(torsten_test_tarball_url)) {
+  torsten_test_tarball_url <- NULL
+}
```

Comment:
This came from `cmdstanr`. I think the idea is to allow the tester to use `TORSTEN_TEST_TARBALL_URL` to test specific versions that are not otherwise exercised by default in the tests. I just adapted it without thinking too hard about it.

Reply:
Okay, fine with me if you want to keep it around.